parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -53,6 +54,7 @@ pub type SyncCommitteeUpdate =
 pub type FinalizedHeaderUpdate =
 	primitives::FinalizedHeaderUpdate<SYNC_COMMITTEE_SIZE, SYNC_COMMITTEE_BITS_SIZE>;
 pub type SyncCommittee = primitives::SyncCommittee<SYNC_COMMITTEE_SIZE>;
+pub type PreparedSyncCommitteeOf = PreparedSyncCommittee<ConstU32<{ SYNC_COMMITTEE_SIZE as u32 }>>;
```

Comment:
Lets use this naming instead:
```suggestion
pub type SyncCommitteePrepared = SyncCommitteePrepared<ConstU32<{ SYNC_COMMITTEE_SIZE as u32 }>>;
```